src/handlers/query.ts

```diff
@@ -151,16 +151,22 @@ async function executeSingleQuery(
   if (query.where) {
     const searchTextFilter = getSearchTextFilter(query.where);
 
-    if ((searchTextFilter.length > 0) && isNearTextFilter(query.where)) {
-      getter.withNearText({
-        concepts: searchTextFilter,
-      });
-    } else if ((searchTextFilter.length > 0) && isMatchTextFilter(query.where)) {
-      getter.withBm25({
-        query: searchTextFilter.toString(),
-      });
+    if (searchTextFilter.length > 0) {
+      if (isTextFilter(query.where, "near_text")) {
+        getter.withNearText({
+          concepts: searchTextFilter,
+        });
+      } else if (isTextFilter(query.where, "match_text")) {
+        getter.withBm25({
+          query: searchTextFilter.toString(),
+        });
+      } else if (isTextFilter(query.where, "hybrid_match_text")) {
+        getter.withHybrid({
+          query: searchTextFilter.toString(),
+        });
```

Comment:
🤔 Is there anyway for us to customize the `alpha` parameter of the hybrid search? Not sure we need it with the current plan (semantic & keyword displayed separately), but would be nice to be able to fine-tune.

Reply:
I was planning to handle `alpha` and `fusionType` as a follow up PR.